### PR TITLE
Accessibility changes: Adding aria tags to improve screen reader performance

### PR DESCRIPTION
**Pull request by c-harper**
Accessibility  updates from a 3rd party review of trimet.org.

### CODE REVIEW

File: packages/location-field/package.json [hunk 1]
```diff
@@ -14,7 +14,13 @@
     "@opentripplanner/geocoder": "^1.1.2",
     "@opentripplanner/humanize-distance": "^1.1.0",
     "@opentripplanner/location-icon": "^1.4.0",
+    "@opentripplanner/core-utils": "^4.1.0",
+    "@opentripplanner/geocoder": "^1.1.0",
+    "@opentripplanner/humanize-distance": "^1.1.0",
+    "@opentripplanner/location-icon": "^1.3.0",
```
Comment (miles-grant-ibigroup) [inline packages/location-field/package.json:20]: Why were some of these packages downgraded?

File: packages/location-field/src/index.js [hunk 2]
```diff
@@ -74,7 +75,8 @@ class LocationField extends Component {
       menuVisible: false,
       message: null,
       geocodedFeatures: [],
-      activeIndex: null
+      activeIndex: null,
+      listBoxId: `listbox-${uniqueId()}`
```
Comment (miles-grant-ibigroup) [inline packages/location-field/src/index.js:79]: There is already an `optionKey` variable which is being used to set a unique key for react. I would suggest either re-using that for the aria-id if possible, or replacing that variable with the new listBoxId.

**Review by miles-grant-ibigroup — CHANGES_REQUESTED** [review]
Thanks so much for adding accessibility to this function! These are long-needed changes.

I only have a few small questions about how the keys are generated as well as what's going on in the dependencies. 

Thanks again for your changes!

**Review by binh-dam-ibigroup — APPROVED** [review]
Conditional approval subject to sorting props where indicated. I favor using the proposed method to generate the id for aria properties.

File: packages/location-field/src/index.js (hunk 2 above)
Comment (c-harper) [reply packages/location-field/src/index.js]: I switched it up to use the optionKey. It works fine as can be seen in storybook and the snapshot tests. Always better to not introduce another dependency.

File: packages/location-field/package.json (hunk 1 above)
Comment (c-harper) [reply packages/location-field/package.json]: I had run a lerna command that I really didn't understand what it was supposed to do and i didn't catch that it added these duplicate refs with lower version numbers. Fixed now.

**Review by miles-grant-ibigroup — APPROVED** [review]
Thanks for all your changes! This is looking great

**Pull request merged**